### PR TITLE
Create task to email user and delete exam

### PR DESCRIPTION
**Pull request by yongsokheng**
- Email to user when user create Exam and not start after 2h
- Delete exam when user create Exam and not start after 24h

### CODE REVIEW

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "reply@example.com"
+  layout 'mailer'
```
Comment (RathanakSreang) [inline app/mailers/application_mailer.rb]: use `"`

File: app/mailers/user_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class UserMailer < ApplicationMailer
+  def inform_user exam
+    @exam = exam
+    mail to: exam.user.email, subject: t("messages.subject")
```
Comment (RathanakSreang) [inline app/mailers/user_mailer.rb]: remove ()

**Comment by HorNorin** [comment]
ok

**Comment by RathanakSreang** [comment]
Ok

**Pull request merged**